ngraph/core/shape_inference/include/tile_shape_inference.hpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+#include <openvino/op/tile.hpp>
+
+#include "shape_infer_utils.hpp"
+namespace ov {
+namespace op {
+namespace v0 {
+
+template <class T>
+void shape_infer(const Tile* op, const std::vector<T>& input_shapes, std::vector<T>& output_shapes) {
+    NODE_VALIDATION_CHECK(op, input_shapes.size() == 2 && output_shapes.size() == 1);
+    const auto& arg_shape = input_shapes[0];
+    const auto& repeats_shape = input_shapes[1];
```

Comment:
This is unconventional. Please stick to the approach with input value like in the Reduce operators.